File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    a = 0
    b = 0
    for i in range(len(s)):
        if s[i].lower() == 'p': 
            a += 1
        elif s[i].lower() == 'y':
            b += 1
    if a == b:
        return True
    else:
        return False

File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        int a = 0;
        int b = 0;
        String string = s.toLowerCase();
        for(int i=0; i<s.length(); i++){
            if(string.charAt(i) == 'p'){
                a++;
            } else if(string.charAt(i) == 'y'){
                b++;
            }
        }
        if(a == b){
            return true;
        } else{
            return false;
        }
    }
}